### DISCUSSION TITLE
statr menu doesent show menu windows 10/11

### DISCUSSION THREAD

**Discussion by aterranegra** [valinet/ExplorerPatcher#3642] [Q&A]
Hi guys,
i installed and it was all ok.
now explorer patcher doesnt show menu win10/win11 in start options...
i cant show my menu in full screen.
how is possible?
tnx...
alessandro
![Senza titolo](https://github.com/user-attachments/assets/dd9f44f9-412f-4308-ace7-715d2420673c)

**Comment by milestprower92** [comment]
Install the latest pre release:
https://github.com/valinet/ExplorerPatcher/releases/tag/22621.3880.66.5_5094108

**Reply by Phoenixshock** [reply]
I was having the same issue and the preview build brought the option and the start menu back. 

![Screenshot - 9_5_2024 , 11_08_24 PM](https://github.com/user-attachments/assets/0e75deb4-1cf0-4396-b184-9472ea05e276)

**Reply by oid23462** [reply]
My browser and computer won't let me download, it says there's a virus and I'm scared

**Reply by pyrates999** [reply]
Read the release notes for EP.  It's not a virus, it's MS claiming it's a hack tool on purpose.  And all you can do exclude the directories that EP gets installed to, and the download directory that EP is downloaded to.

**Reply by IgnoreMyname** [reply]
Thank you! Back to having the Win 10 start menu!

**Reply by aterranegra** [reply]
> Install the latest pre release: https://github.com/valinet/ExplorerPatcher/releases/tag/22621.3880.66.5_5094108

i stopped fucking mc afee and insalled as admin...
it works... i have my full screen menu with tiles...

tanx so mutch!!!
Alessandro

**Comment by CrazyAL01** [comment]
So I'm a little frustrated too.  Since I think a few weeks, the start menu appears like the Windows 11 style menu (WHICH I HATE).  I love EP, but looks like some Win 11 update may have screwed me up.  I do have the latest EP installed.  Actually when I noticed the menu had reverted back to the Win 11 format I also noticed my START button was gone and the systray had a square around it.  I uninstalled and reinstalled EP and all (except the start menu) went back to normal.  Can anyone help me in regards to the start menu?  I want to have back the Win 10 style TILE menu.
Thanks

**Reply by pyrates999** [reply]
Install latest release: https://github.com/valinet/ExplorerPatcher/releases

Read the release notes on what exclusions to add to AV before downloading and installing.

**Reply by CrazyAL01** [reply]
Hi there.  Well while I applaud your effort to help, I am unfortunately lost to understanding what it is I need to do.  I DID however enter all the exclusions on Windows security so as not to have problems for future installs.   But I don't quite know what to do regarding the release notes to fix the menu.  incidentally I have version 23H2

**Reply by CrazyAL01** [reply]
ok THIS is weird.  Looks like after I reinstalled, the settings went back to displaying a WIn 11 menu.  I just changed it to Win 10 and ALL working perfectly now!  YAY!!!

**Reply by pyrates999** [reply]
> Hi there. Well while I applaud your effort to help, I am unfortunately lost to understanding what it is I need to do. I DID however enter all the exclusions on Windows security so as not to have problems for future installs. But I don't quite know what to do regarding the release notes to fix the menu. incidentally I have version 23H2

The latest release fixes the start menu not showing.

**Reply by CrazyAL01** [reply]
yep it's all working now!!  YAY!
TY TY!
